import-service/serverless.yml

```diff
@@ -12,6 +12,11 @@ provider:
                 -   "s3:*"
             Resource:
                 -   "arn:aws:s3:::uploaded-mp/*"
+        -   Effect: "Allow"
+            Action: 
+                -   "sqs:*"
+            Resource:
+                -   "arn:aws:sqs:eu-west-1:869906325472:mp-aws-sqs-queue"
```

Comment:
Please export SQSQueue from products service and import here

Reply:
Is this a good way to import some data from another service? Looks strange if we will start to import data higher then root holder of each service.